### PR TITLE
Add Sentry

### PR DESCRIPTION
**Pull request by raccube**

### CODE REVIEW

File: helpdesk/helpdesk/settings.py [hunk 1]
```diff
@@ -39,6 +40,12 @@
         ) from None
     raise
 
+sentry_sdk.init(
+    dsn=getattr(configuration, "SENTRY_DSN", None),
+    traces_sample_rate=getattr(configuration, "SENTRY_TRACES_SAMPLE_RATE", 1.0),
+    profiles_sample_rate=getattr(configuration, "SENTRY_PROFILES_SAMPLE_RATE", 1.0),
```
Comment (RealOrangeOne) [inline helpdesk/helpdesk/settings.py:46]: Tracing and profiling jumps the usage quota a fair bit, and for this project isn't really something we care about. Probably best leaving these unset

Comment (raccube) [reply helpdesk/helpdesk/settings.py:46]: We can just set it to 0

Comment (raccube) [reply helpdesk/helpdesk/settings.py]: b219f7d

**Review by RealOrangeOne — APPROVED** [review]
If CI is happy, then I'm happy.

**Pull request merged**